lib/msf/core/constants.rb

```diff
@@ -49,6 +49,49 @@ module Msf
     ExcellentRanking => "excellent"
   }
 
+#
+# Stability traits
+#
+
+# Module does not crash the service
+CRASH_SAFE             = 'crash-safe'
+# Module crashes service, but service restarts.
+CRASH_SERVICE_RESTARTS = 'crash-service-restarts'
+# Module crashes service, and service remains down.
+CRASH_SERVICE_DEAD     = 'crash-service-dead'
+# Module crashes the OS, but OS restarts.
+CRASH_OS_RESTARTS      = 'crash-os-restarts'
+# Module crashes the OS, and OS remains down.
+CRASH_OS_DEAD          = 'crash-os-dead'
```

Comment:
This makes it sound like the system is bricked. Maybe "down" is better than "dead."